### DISCUSSION TITLE
GitHub Benefits for Students

### DISCUSSION THREAD

**Discussion by zwid3** [community/community#138439] [GitHub Education]
### Hi everyone,

I am currently waiting to receive my Git Education benefit: Student Developer Pack. I have not received the email with benefits. I was approved 4 days ago. Any assistance will be greatly appreciated.

**Comment by davevad93** [comment]
Hi @BrianNdhlovu ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @BrianNdhlovu was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by masterwww1** [comment]
You are currently waiting to receive your GitHub Education benefit, specifically the Student Developer Pack. You were approved for the program 4 days ago but have not yet received the email containing the details or access to the benefits. You have checked your inbox, including spam and promotions folders, but haven’t found any correspondence.

Could you please assist in resolving this or let me know if there is any further action you need to take to receive the pack? Your help and guidance in addressing this would be greatly appreciated. Thank you in advance for your support.

**Comment by ElisavetAmpatzidou** [comment]
Hi, I have a similar problem. My status is "approved" since 12 January, but in my profile, my current plan is "GitHub Free" and I don't have access to pro features. 
![Screenshot 2024-09-12 125936](https://github.com/user-attachments/assets/dfe931c7-a641-4a0d-8ddf-317097f0e794)
Does it affect that my academic email is not the primary email of my account?
 
Note: I have renew my student account because it was expired.

**Reply by davevad93** [reply]
Hi @ElisavetAmpatzidou , 

You should have access to your benefits indeed. Set your academic email as the primary one, then go to the [pack offers page](https://education.github.com/pack) and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.
